src/Makefile.am

```diff
@@ -2,8 +2,10 @@
 EXTRA_DIST = \
 	.indent.pro
 
-ubindir = ${prefix}/bin
-usbindir = ${prefix}/sbin
+prefix = /usr/local
```

Comment:
Are you sure this is right? I don't think we're supposed to set these at all. AFAIK, automake sets them for us.

Reply:
https://www.gnu.org/software/autoconf/manual/autoconf-2.72/html_node/Default-Prefix.html

Reply:
No, I'm not sure at all.  I know those are the values we want, but I didn't know at all how they should be defined in autotools.  As they were being defined, I thought I had to define them somewhere.  I guess I'll just remove them.

Thanks!

Reply:
I've removed it now; please check if it makes sense.